### PR TITLE
New icons: user-search and user-round-search

### PR DESCRIPTION
**Pull request by jmsv**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

New icons:
- `user-search`
- `user-search-2`

### Icon use case 

- Searching for users
- Recruitment

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: `user-check`, `user-plus`, `user-x` etc. + corresponding `-2` variations, plus `search`
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by jguddas** [comment]
icon (21)
https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8Y2lyY2xlIGN4PSIxMCIgY3k9IjciIHI9IjQiIC8+CiAgPGNpcmNsZSBjeD0iMTciIGN5PSIxNyIgcj0iMyIgLz4KICA8cGF0aCBkPSJtMjEgMjEtMS45LTEuOSIgLz4KICA8cGF0aCBkPSJNMTAuMyAxNUg3YTQgNCAwIDAgMC00IDR2MiIgLz4KPC9zdmc+=

**Comment by jmsv** [comment]
@jguddas that looks good, did that exist somewhere already? missed it if so

had a go at making a similar version for the `user-2` style:

icon (1)

**Comment by jmsv** [comment]
updated this PR with the new icons, looks much better, thanks for having a look!

File: icons/user-search.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jmsv"
```
Comment (jguddas) [inline icons/user-search.json:4]:
```suggestion
    "jmsv",
    "jguddas",
    "colebemis",
    "csandman",
    "ericfennis",
    "karsa-mistmere"
```

File: icons/user-search-2.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jmsv"
```
Comment (jguddas) [inline icons/user-search-2.json:4]:
```suggestion
    "jmsv",
    "karsa-mistmere"
```

**Review by jguddas — CHANGES_REQUESTED** [review]
It's based on an existing icon, so it should inherit the contributors.

**Comment by jmsv** [comment]
> It's based on an existing icon, so it should inherit the contributors.

gotcha, missed that – fixed

**Review by jguddas — COMMENTED** [review]
`user-search-2` violates naming guidelines, icons should not end with `-2`.

I highly recommend creating one PR per icon.

**Comment by jguddas** [comment]
IMO adding icons with the same use case but with different styles is something we should avoid, it works against having one consistent feel across all icons.

So 👍 for `user-search` and 👎 for `user-search-2` from me.

**Comment by jmsv** [comment]
Thanks for having a look @jguddas and @karsa-mistmere ❤️ 

---

> I highly recommend creating one PR per icon.

From [CONTRIBUTING.md](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md):

> If you want submit multiple icons, please separate the icons and group them. That makes reviewing the icons easier and keep the thread clean and scoped. So don't submit multiple icons in one PR that have noting to do with each other. So for example don't create one PR with icons: arrow-up, bicycle, arrow-down. Seperate them by two PRs; 'pr-01' arrow, arrow-down and 'pr-02' bicycle.

As-per these guidelines I think I was correct in making this one PR.

---

Re the `user*-2` naming I'm aware that the [Naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions) advise against `-2` for "what makes the alternate unique" but I opted for naming consistency instead – I agree with @karsa-mistmere that it makes more sense than deviating.

I like the sound of `user-round-*`! Should take into account though that this would be a breaking change for anyone currently using `user*-2` icons.

---

As for what to do with this PR, it looks like the options are:

1. Keep the `user-search-2` name for now
2. Rename `user-search-2` to `user-round-search`
3. Remove `user-search-2` entirely

Option 2 isn't ideal since it introduces naming inconsistency, and option 3 isn't ideal since it _also_ introduces inconsistency considering every other `user-*` icon has a `-2` variant.

Hoping we can go with option 1 but let me know what you think

**Comment by karsa-mistmere** [comment]
We can add backwards compatible aliases for current `user-*-2` icons.

**Comment by jmsv** [comment]
> We can add backwards compatible aliases for current `user-*-2` icons.

Ah, awesome! I'd be happy to have a go at setting that up

**Comment by jguddas** [comment]
> > We can add backwards compatible aliases for current `user-*-2` icons.
> 
> Ah, awesome! I'd be happy to have a go at setting that up

If you are up for it, you can give my new `renameIcon.sh` script a try.

https://github.com/lucide-icons/lucide/pull/1630

## How to use it

Make sure you have the [gh cli](https://cli.github.com/) installed.

Don't think this will work on Windows.

```bash
gh pr checkout 1630 # go to my PR with the new rename script
./scripts/renameIcon.sh user-2 user-round # run the script
git checkout main # go back to main
git checkout -b rename/user-2-to-user-round # create a new branch
git commit -m 'Renamed user-2 to user-round' # commit the changes
gh pr create --title 'Renamed user-2 to user-round' # create a pull request
git checkout main # go back to main
```

**Comment by jmsv** [comment]
Done! #1638

**Review by ericfennis — APPROVED** [review]

**Pull request merged**